config/common.go

```diff
@@ -35,3 +43,13 @@ func GetMinimumQuestionsCountPerShard(status *guard.FileStoreStatus) (val int) {
 	//return int(status.FileStoreMeta.CheckFrequency) * ConstMinQuestionsCountPerChallenge
 	return ConstMinQuestionsCountPerShard
 }
+
+func InitDB() {
```

Comment:
Name is confusing, maybe something like `initTestDB`

Reply:
ok.